docs/REST API/rest-api.md

```diff
@@ -9138,6 +9141,125 @@ Add App Store (VPP) app purchased in Apple Business Manager.
 
 `Status: 200`
 
+### List Fleet library apps
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+List available Fleet library apps.
+
+`GET /api/v1/fleet/software/fleet_library_apps`
```

Comment:
### Dev note

Return Fleet library apps that haven't been added to a team yet. (same behaviour as for VPP apps).

Reply:
```suggestion
`GET /api/v1/fleet/software/library`
```
I would recommend less words imo. Take it or leave it.